contracts/root/RootChainManager/RootChainManager.sol

```diff
@@ -41,12 +40,13 @@ contract RootChainManager is IRootChainManager, Initializable, AccessControl {
         _;
     }
 
-    // TODO: add fallback function
+    receive() external payable {
+        _depositEtherFor(_msgSender());
+    }
```

Comment:
what if someone sends eth with some call data (calling a function that does not exist)? should we add `fallback()` for that case or will receive work?

Reply:
IMO if someone is sending eth with random call data means he is doing something we don't expect him to do so its is fine to fail in that case.

Reply:
cool approving the pr